pom.xml

```diff
@@ -5,7 +5,7 @@
     <groupId>org.sagebionetworks.bridge</groupId>
     <artifactId>sdk-group</artifactId>
     <!-- Please use the maven versions plugin to manage this, e.g.`mvn versions:set -DnewVersion=0.8.1`-->
-    <version>0.12.21</version>
```

Comment:
Not sure why it's showing 0.12.21 as the old version. 0.12.22 appears to already be in develop, could you bump to 0.12.23?